templates/tfengine/components/devops/terraform.tfvars

```diff
@@ -11,17 +11,16 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-{{$missing_admins_group := not (get .admins_group "exists")}}
 admins_group = {
+  customer_id  = "{{get . "admins_group.customer_id" ""}}"
+  description  = "{{get . "admins_group.description" ""}}"
+  {{- $default_admins_group_display_name := regexReplaceAll "@.*" (get . "admins_group.id" "") ""}}
+  display_name = "{{get . "admins_group.display_name" $default_admins_group_display_name}}"
+  exists       = {{get .admins_group "exists" false}}
   id           = "{{.admins_group.id}}"
-  {{if $missing_admins_group -}}
-  display_name = "{{get .admins_group "display_name" (regexReplaceAll "@.*" .admins_group.id "")}}"
-  customer_id  = "{{.admins_group.customer_id}}"
-  {{hclField .admins_group "description" -}}
-  {{hclField .admins_group "owners" -}}
-  {{hclField .admins_group "managers" -}}
-  {{hclField .admins_group "members" -}}
-  {{- end -}}
+  owners       = [{{range get . "admins_group.owners"}}"{{.}}",{{end}}]
+  managers       = [{{range get . "admins_group.managers"}}"{{.}}",{{end}}]
+  members       = [{{range get . "admins_group.members"}}"{{.}}",{{end}}]
```

Comment:
For style, I think the one-liner version here is less readable than the expanded multi-line version, I would prefer you use the latter.

Reply:
You're right, I was kinda unsure wether to use this one or the multiline. For empty cases, the multiline ends up being like:

```
owners = [
]
```

I changed it to multiline